src/main/java/stsjorbsmod/cards/cull/deckoftrials/Blasphemer.java

```diff
@@ -19,8 +19,8 @@
     public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
 
     private static final int COST = COST_UNPLAYABLE;
-    private static final int SMITE_AMOUNT = 5;
-    private static final int UPGRADE_SMITE_AMOUNT = 1;
+    private static final int SMITE_AMOUNT = 4;
+    private static final int UPGRADE_SMITE_AMOUNT = 2;
```

Comment:
Do we know if he wanted 4(5) or 4(6)?

Reply:
He only mentioned that 5 should come down to 4, so I assumed he still wanted Blasphemer+ to be 6 still. Probably should clarify.